styles.css

```diff
@@ -749,6 +754,149 @@ input:hover {
   font-size: 20px;
 }
 
+/* popup window styles */
+.popupOutline {
+  background: #c1c7d0;
+  top: 0;
+  left: 0;
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 98;
+}
+
+.pop-window {
+  border-radius: 8px;
+  background-color: white;
+  width: 85%;
+  height: 92%;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  top: 50%;
+  left: 50%;
+  position: absolute;
+  overflow-y: scroll;
+  transform: translate(-50%, -50%);
+  transition: 0.2s ease-in-out;
+  z-index: 99;
+  gap: 12px;
+}
+
```

Comment:
Hello @tan12082001, You did well with the popup menu..well-done 👍 
Please id suggest that for your desktop view, you increase the height of the popup screen bit to look similar to the Figma template you were given and also use the right text alignment for the heading of the popup so your design looks better 😄 

Figma design :
![image](https://github.com/tan12082001/Project-portfolio/assets/20537712/1fb4b0ad-89f9-437a-a58b-07c42d64e7c2)

Your design :
![image](https://github.com/tan12082001/Project-portfolio/assets/20537712/55bf7188-78d3-4575-a7c0-10d7ada5fc84)